### PR TITLE
New python interfaces: policy support

### PR DESCRIPTION
**Pull request by michael-smirnov**
- Support of device offload using old `daal4py.oneapi.sycl_context` control
- Fix of data transfer from oneDAL to Python

### CODE REVIEW

File: onedal/common/backend/policy.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include "common/backend/policy.h"
+
+namespace oneapi::dal::python {
+
+policy::policy(const std::string& device_name) {
+    if (device_name == "host") {
+        impl_ = std::shared_ptr<policy_impl>(new host_policy_impl());
+    }
+#ifdef ONEDAL_DATA_PARALLEL
+    else if (device_name == "gpu") {
```
Comment (PetrovKP) [inline onedal/common/backend/policy.cpp:26]: And for a cpu device?

**Review by PetrovKP — APPROVED** [review]
Merge after successful CI. I hope the ID pull request will not affect your changes)))

**Comment by napetrov** [comment]
@samir-nasibli  @Alexsandruss  - is it worth looking on this now?

**Pull request closed without merging**